lib/cmake-coverage

```diff
@@ -1 +1 @@
-Subproject commit fde8c1ef14684d763e7728dda98d49f2330534d1
+Subproject commit 25af3c657e42543e3e8b43378ebc5381e6260daa
```

Comment:
I added a filter for the lib folder. We don't need coverage reports for the libs